taxi/models.py

```diff
@@ -15,7 +16,13 @@ def __str__(self):
 
 
 class Driver(AbstractUser):
-    license_number = models.CharField(max_length=255, unique=True)
+    license_number = models.CharField(
+        max_length=8,
+        unique=True,
+        validators=[validate_license_number],
+        blank=False,
+        null=False
+    )
```

Comment:
The `license_number` field in the `Driver` model is correctly defined with a validator. Ensure that the `validate_license_number` function is implemented to check for the format of 3 uppercase letters followed by 5 digits, as required by the task.